IrinaLoginovaHomeWork_3_1.cpp

```diff
@@ -0,0 +1,17 @@
+#include <cstdint>
+#include <cstring>
+#include <iostream>
+
+int main() {
+  constexpr int n = {150};
+  char ArrayString[n] = {};
```

Comment:
```suggestion
  char ArrayString[n] {};
```
не нужно каждый раз при создании переменной использовать оператор `=`